pywbemtools/pywbemcli/_cmd_profile.py

```diff
@@ -0,0 +1,252 @@
+# (C) Copyright 2017 IBM Corp.
+# (C) Copyright 2017 Inova Development Inc.
+# All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Click command definition for the profile command group which includes
+cmds for inspection and management of the registered profiles that are
+maintained in the WBEM Server and used by the client to determine what
+WBEM management functionality the server supports.
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import click
+
+from pywbem import ValueMapping, Error
+
+from .pywbemcli import cli
+from ._common import format_table, raise_pywbem_error_exception, \
+    validate_output_format, CMD_OPTS_TXT, GENERAL_OPTS_TXT, SUBCMD_HELP_TXT
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+# NOTE: A number of the options use double-dash as the short form.  In those
+# cases, a third definition of the options without the double-dash defines
+# the corresponding option name, ex. 'include_qualifiers'. It should be
+# defined with underscore and not dash
+
+
+@cli.group('profile', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def profile_group():
+    """
+    Command group for WBEM server management profiles.
+
+    This command group defines commands to inspect and manage the WBEM
+    management registered profiles maintained by the WBEM server.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'server' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@profile_group.command('list', cls=PywbemcliCommand,
+                       options_metavar=CMD_OPTS_TXT)
+@click.option('-o', '--organization', type=str, metavar='ORG-NAME',
+              required=False,
+              help='Filter by the defined organization. (ex. -o DMTF')
+@click.option('-p', '--profile', type=str, metavar='PROFILE-NAME',
+              required=False,
+              help='Filter by the profile name. (ex. -p Array')
+@add_options(help_option)
+@click.pass_obj
+def profile_list(context, **options):
+    """
+    List management profiles advertised by the server.
+
+    Retrieve the CIM instances representing the WBEM management profiles
```

Comment:
We should not say that it retrieves the CIM instances, instead let's talk about "listing the WBEM management profiles"

Reply:
DONE